doc/Command_Index_internal.tex

```diff
@@ -1515,7 +1515,10 @@ \subsection{Flags}
 Boolean which indicates that a hyphen after an empty first syllable should be forced.
 
 \macroname{\textbackslash ifgre@showhyphenafterthissyllable}{}{gregoriotex-syllable.tex}
-Boolean used by \verb=\GreSyllable= to decide if a hyphen should be shown after the syllable.
+Boolean set and used by \verb=\GreSyllable= to decide if a hyphen should be shown after the syllable.
+
+\macroname{\textbackslash ifgre@possibleluahyphenafterthissyllable}{}{gregoriotex-syllable.tex}
+Boolean set by \verb=\GreSyllable= indicating if the lua pass may add an hyphen after the syllable, used by \verb=\gre@calculate@eolshift= for protrusion calculation.
```

Comment:
*Very minor*, but Lua should be capitalized, as it is in `Command_Index_User.tex`.